port_ocean/clients/port/mixins/integrations.py

```diff
@@ -50,13 +51,36 @@ async def get_log_attributes(self) -> LogAttributes:
             self._log_attributes = response["logAttributes"]
         return self._log_attributes
 
+    async def _poll_integration_until_default_provisioning_is_complete(
+        self,
+        interval=15,
+    ) -> Dict[str, Any]:
+        response = await self._get_current_integration()
+        response_json = response.json()
+        config = response_json.get("integration", {}).get("config", {})
+        if config != {}:
+            return response_json
+
+        logger.info(
+            f"Still waiting for integration config to be ready, waiting {interval} seconds"
+        )
```

Comment:
I would think of a different phrasing for this log, I am not a fan of the "still waiting" terminology

Reply:
Fixed!